src/frontend/src/lib/components/send/SendModal.svelte

```diff
@@ -64,6 +65,9 @@
 		};
 		await loadTokenAndRun({ token, callback });
 	};
+
+	let source: string;
+	$: source = $icrcAccountIdentifierText ?? '';
```

Comment:
We should add a TODO here right as the source won't remain used as `$icrcAccountIdentifierText` in this component? Do I get it right.

Reply:
That is right. @DenysKarmazynDFINITY should take care of it when implementing sending bitcoin. I wanted to keep the same logic in this refactor. I added a comment